portfolio/src/main/webapp/map.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Chenyu's portfolio</title>
+
+    <!-- Font awesome icons. -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <!-- Bootstrap CSS. -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    
+    <link rel="stylesheet" href="style.css">
+
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBYoh-rlQfT-0wDveS2mahHXGdKaO8Plbg"></script>
+    <script src="script-map.js"></script>
+  </head>
+  <body onload="createMap()">
+    <div id="map"></div>  
```

Comment:
I would rename this id to something a little more specific to say what kind of map this is/where it is used.
Reasoning: You could have multiple maps on your site.

Reply:
got it